auth-web/src/components/auth/DecideBusinessInfo.vue

```diff
@@ -0,0 +1,74 @@
+<template>
```

Comment:
is it more of a view or a component? If its a view , I would suggest to move this to views folder

Reply:
Thats a good point. In this instance, these components are more like Views than functional components. I will restructure this, thanks!

Reply:
Updated in next commit 👍